server/cla.go

```diff
@@ -37,9 +37,18 @@ func (s *Server) checkCLA(pr *model.PullRequest) {
 	}
 
 	username := pr.Username
-	mlog.Info("Will check the CLA for user", mlog.String("user", username),
-		mlog.String("repo", pr.RepoOwner), mlog.String("reponame", pr.RepoName),
-		mlog.Int("pr n", pr.Number))
+	mlog.Info(
+		"Will check the CLA for user",
+		mlog.String("user", username),
+		mlog.String("repo", pr.RepoOwner),
+		mlog.String("reponame", pr.RepoName),
+		mlog.Int("pr n", pr.Number),
+	)
+
+	if excluded := isUserExcluded(s.Config.CLAExclusionsList, username); excluded {
```

Comment:
You can simplify this to
```suggestion
	if isUserExcluded(s.Config.CLAExclusionsList, username) {
```

Reply:
Yeah that was some leftover, originally I had it like `exlucded, err := isUserExcluded(...)` but dropped the `error` and forgot about this.